src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -252,4 +252,13 @@
   <data name="InvalidConstraint" xml:space="preserve">
     <value>Constraint is not supported for this type and will be ignored.</value>
   </data>
+  <data name="NonAppJsonTypeNotSupported" xml:space="preserve">
+    <value>Non application json MIME type has limited support</value>
+  </data>
+  <data name="SupportedSchemesWarningMessage" xml:space="preserve">
+    <value>Non-HTTPS schemes have limited Autorest support</value>
+  </data>
+  <data name="NetworkInterfacesApiWarningMessage" xml:space="preserve">
+    <value>Swagger API can reference either Microsoft.Compute or Microsoft.Network Apis and not both</value>
```

Comment:
`Swagger` is deprecated. The new term is `OpenAPI` ... but maybe we should drop the reference to the format, and say something like `May not reference more than one of {"Microsoft.Compute","Microsoft.Network"}` 

- this makes it so we can add another item to the list without substantially changing the message later.